content/applications/sales/sales/products_prices/taxes.rst

```diff
@@ -1,14 +0,0 @@
-:nosearch:
-
-=========
-Set taxes
-=========
-
-.. toctree::
-   :titlesonly:
-
-   /applications/finance/accounting/taxation/taxes/default_taxes
-   /applications/finance/accounting/taxation/taxes/fiscal_positions
-   /applications/finance/accounting/taxation/taxes/taxes
-   /applications/finance/accounting/taxation/taxes/taxcloud
-   /applications/finance/accounting/taxation/taxes/B2B_B2C
```

Comment:
Hi @odoo/sales-doc-review 

If (when) this PR goes through, it will remove this double indexation of the pages about taxes (normally, we shouldn't have the same doc twice in the toc tree, but this was done a long while ago). However, we should replace this with a paragraph somewhere that explain to the sales users that there are docs to read about taxes, with a seealso section.